lib/db/seed_file_loader.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+# Copyright (c) 2015 The Regents of the University of Michigan.
+# All Rights Reserved.
+# Licensed according to the terms of the Revised BSD License
+# See LICENSE.md for details.
+
+require "json"
+
+module Db
+
+  # Loads seed files and converts to a ruby hash.
+  class SeedFileLoader
+    BASE_DIR = File.join "db", "seeds"
+
+    attr_reader :seed_dir
+    def initialize(env_name, base_dir = BASE_DIR)
+      @seed_dir = File.join(base_dir, "#{env_name}-seeds.d")
```

Comment:
The case where there are no seed files is not exceptional.  Currently, seed_files on a non-extant dir will return `[]`, and this code properly does nothing.